src/App.js

```diff
@@ -1,16 +1,30 @@
 import React, { Component } from "react";
-
+import Heading from './components/Heading'
 import Bookings from "./Bookings";
 import Restaurant from "./Restaurant";
 import "./App.css";
+import TouristInfoCards from "./components/TouristInfoCards";
+import Footer from "./components/Footer";
 
 class App extends Component {
   render() {
     return (
       <div className="App">
-        <header className="App-header">CYF Hotel</header>
+        <Heading />
         <Bookings />
         <Restaurant />
+        <div className="card">
+          <TouristInfoCards src="1" alt="London" city="London" />
```

Comment:
This isn't really wrong, but the name `TouristInfoCards` implies a *plural* (because of the "s" on the end) but only 1 card is being shown.

I would rename your existing component to be called `TouristInfoCard` (notice there's no "s") and then create a new component `TouristInfoCards` which has the `<div className="card">` and all 3 cards inside it.